src/SpringReverb.cpp

```diff
@@ -123,17 +123,16 @@ struct SpringReverb : Module {
 
 		// Set lights
 		float lightRate = 5.0 * args.sampleTime;
-		vuFilter.setRate(lightRate);
-		vuFilter.process(std::fabs(wet));
-		lightFilter.setRate(lightRate);
-		lightFilter.process(std::fabs(dry*50.0));
-
-		float vuValue = vuFilter.peak();
+		vuFilter.setLambda(1.f - lightRate);
+		float vuValue = vuFilter.process(1.f, std::fabs(wet));
+		lightFilter.setLambda(1.f - lightRate);
+		float lightValue = lightFilter.process(1.f, std::fabs(dry * 50.0));
+		
```

Comment:
Replacing all this with `dsp::VuMeter2` would be good here. That class was written in ~2018 after this code in ~2017.

Reply:
I've updated this, but I'm a little unsure about what the correct behaviour should be, especially given befaco VU labels are actually DBu and Vpp. I don't think I've got it quite right yet.

**current**
![reverb_current](https://user-images.githubusercontent.com/915048/124794306-c7ef8500-df46-11eb-8925-d80a09a3d46b.gif)

**old**
![reverb_previous](https://user-images.githubusercontent.com/915048/124794281-befeb380-df46-11eb-8737-99f426192b97.gif)

I've seen your response on the forum, so will have another think but if it's obvious to you what the solution should be then let me know.

Reply:
I like the first one better since it's actually accurate. My only suggestion is to make the 10Vpp red light not turn on immediately but gradually. For example you have
```cpp
lights[0].setBrightness(vuMeter.getBrightness(0.f, 0.f))
```
set it to
```cpp
lights[0].setBrightness(vuMeter.getBrightness(-3.f, 0.f))
```
or whatever the next light's max dB is. I think that'll match what my hardware unit does.

Reply:
Yup good idea - I've made the change, and seems to work nicely (now you can basically start to hear clipping as you transition into the red).